service/src/main/java/uk/gov/hmcts/reform/fpl/enums/DocmosisTemplates.java

```diff
@@ -1,8 +1,9 @@
 package uk.gov.hmcts.reform.fpl.enums;
 
 public enum DocmosisTemplates {
-    C6("FL-PLW-HNO-ENG-00092.doc", "Notice_of_proceedings_c6.pdf"),
-    C6A("FL-PLW-HNO-ENG-00093.doc", "Notice_of_proceedings_c6a.pdf"),
+    C110A("FL-PLW-APP-ENG-00095.docx", "case_submission_c110a.pdf"),
+    C6("FL-PLW-HNO-ENG-00092.docx", "Notice_of_proceedings_c6.pdf"),
```

Comment:
This should be C6 & C6A FL-PLW-HNO-ENG-00092.doc and FL-PLW-HNO-ENG-00093.doc, as the .docx versions of these file are no longer used.